src/commands/doctor.ts

```diff
@@ -95,9 +95,24 @@ module.exports = {
       expoVersion = "-"
       expoWorkflow = "not installed"
     }
-
     const expoInfo = [column1("expo"), column2(expoVersion), column3(expoWorkflow)]
 
+    let depExpoCLIInfo = null
+
+    try {
+      const expoCLIVersionOutput = await run("expo-cli --version", { trim: true })
```

Comment:
```suggestion
      const expoCliVersionOutput = await run("expo-cli --version", { trim: true })
```
nit: code consistency here